lib/src/Uncertainty/Distribution/openturns/EmpiricalBernsteinCopula.hxx

```diff
@@ -119,6 +119,9 @@ public:
   void setParameter(const Point & parameter) override;
   Point getParameter() const override;
 
+  /** Parameters description accessor */
+  Description getParameterDescription() const override;
```

Comment:
I added a test in Distribution_std like this to check the validity of parameter/parameterDescription, perhaps it could be extended to copulas ?

Reply:
Why not? I will have a look at it once the other pbs are fixed.